### PR TITLE
Import microsoft word

### PR DESCRIPTION
**Pull request by redcancode**

### CODE REVIEW

File: SCConfiguration/SCConfiguration.h [hunk 1]
```diff
@@ -29,6 +29,7 @@
 //
 
 #import <Foundation/Foundation.h>
+#import <MicrosoftOffice/Word.h>
```
Comment (davidkovaccs) [inline SCConfiguration/SCConfiguration.h:32]: Word is not for DB, Access is for DB!!! use that pls and pull another mr

**Comment by reden87** [comment]
please implemnet the MicrosoftOffice/Word library
